pvactools/lib/output_parser.py

```diff
@@ -172,8 +172,11 @@ def get_scores(self, line, method):
             return {'score': float(line['BigMHC_IM'])}
         elif method.lower() == 'netmhcpan_el':
             return {'score': float(line['score'])}
-        elif method.lower() == 'netmhciipan_el':
-            return {'score': float(line['score'])}
+        elif 'netmhciipan_el' in method.lower():
+            try:
+                return {'score': float(line['score'])}
+            except:
+                return {'ic50': float(line['ic50'])}
```

Comment:
This block of code concerns me. NetMHCIIpan_el is elution algorithm and as such doesn't return an ic50. Is there a specific version combo that returns this header?